paasta_tools/kubernetes_tools.py

```diff
@@ -323,6 +323,8 @@ def _set_disrupted_pods(self: Any, disrupted_pods: Mapping[str, datetime]) -> No
         "yelp.com/paasta_instance": str,
         "yelp.com/paasta_service": str,
         "sidecar.istio.io/inject": str,
+        "yelp.com/pool": str,
```

Comment:
do we need the un-prefixed label on pods? if we don't need it, we might as well skip it as otherwise we'll need another big bounce when we remove it

Reply:
I am keeping it because to keep backward compatibility just in case, in a perfect world I would keep it :)

Reply:
you're not keeping it in this case though, right? you're adding the old/deprecated version.

I agree with @nemacysts here - we can just add the new label

Reply:
> couple of notes - please also squash your commits before merging

Yes.

Reply:
Makes sense @nemacysts @Rob-Johnson I will do the changes 👍